src/cpp/transport/TCPTransportInterface.cpp

```diff
@@ -627,86 +496,95 @@ bool TCPTransportInterface::CloseInputChannel(const Locator_t& locator)
     return bClosed;
 }
 
-void TCPTransportInterface::close_tcp_socket(TCPChannelResource *p_channel_resource)
+//TODO Rename
+void TCPTransportInterface::close_tcp_socket(
+        std::shared_ptr<TCPChannelResource>& channel)
```

Comment:
According to the style guide, one-parameter methods should be declared in one line if doesn't surpass the limit of 120 characters.